pkg/resourcemonitor/noderesourcesaggregator.go

```diff
@@ -152,6 +152,29 @@ func (noderesourceData *nodeResources) Aggregate(podResData []PodResources) topo
 				available:   int64(0),
 				capacity:    int64(len(noderesourceData.reservedCPUIDPerNUMA[nodeID])),
 			}
+
+			for resName, allocatable := range nodeRes {
```

Comment:
this `else` statement means that given NUMA node doesn't exist, not that a resource was absent during NFD updater startup. And the missing resources are already in `perNuma` map, the problem is that they are being rejected in `updateAvailable` func